node/src/components/consensus/era_supervisor.rs

```diff
@@ -1300,8 +1296,30 @@ async fn execute_finalized_block<REv>(
             return;
         }
     }
+
+    // Get the deploy hashes for the finalized block.
+    let transaction_hashes = finalized_block.transfer_hashes().to_owned();
```

Comment:
```suggestion
    // Get the transfer hashes for the finalized block.
    let transfer_hashes = finalized_block.transfer_hashes().to_owned();
```

Reply:
Fixed in https://github.com/casper-network/casper-node/commit/8e89b9c98236e80661953dce42d8328fd93a994f